main.go

```diff
@@ -267,9 +269,9 @@ func createKubeClient(inCluster bool, apiserver string, kubeconfig string) (kube
 	return kubeClient, nil
 }
 
-func metricsServer(registry prometheus.Gatherer, port int) {
+func metricsServer(registry prometheus.Gatherer, host string, port int) {
 	// Address to listen on for web interface and telemetry
-	listenAddress := fmt.Sprintf(":%d", port)
+	listenAddress := fmt.Sprintf("%s:%d", host, port)
```

Comment:
One nit:

`net.JoinHostPort(host, strconv.Itoa(port))` is more preferred, imo.